pythonforandroid/recipe.py

```diff
@@ -859,6 +859,17 @@ class CompiledComponentsPythonRecipe(PythonRecipe):
 
     build_cmd = 'build_ext'
 
+    def get_recipe_env(self, arch=None, with_flags_in_cc=True):
+        env = super(CompiledComponentsPythonRecipe,
+                    self).get_recipe_env(arch, with_flags_in_cc)
+        if not self.call_hostpython_via_targetpython:
+            if 'python2' in self.ctx.recipe_build_order:
+                env['PYTHON_ROOT'] = self.ctx.get_python_install_dir()
+                env['CFLAGS'] += ' -I' + env['PYTHON_ROOT'] + '/include/python2.7'
+                env['LDFLAGS'] += ' -L' + env['PYTHON_ROOT'] + '/lib' + \
+                                  ' -lpython2.7'
```

Comment:
This probably shouldn't hardcode python2.7 due to the python3 build, but I have to think a little about what's supposed to happen.